src/StoreKeeper/WooCommerce/B2C/Helpers/DateTimeHelper.php

```diff
@@ -10,36 +10,40 @@
 class DateTimeHelper
 {
     const MYSQL_DATE_FORMAT = 'Y-m-d H:i:s';
+    const WORDPRESS_DATE_FORMAT_OPTION = 'date_format';
+    const WORDPRESS_TIME_FORMAT_OPTION = 'time_format';
 
     /**
      * Get current date and time
      * Don't use non-GMT time, unless you know the difference and really need to.
      *
      * @throws Exception
      */
-    public static function currentDateTime($gmt = true): DateTime
+    public static function currentDateTime(): DateTime
     {
-        $timezone = $gmt ? new DateTimeZone('UTC') : wp_timezone();
-
-        return new DateTime('now', $timezone);
+        return new DateTime('now', new DateTimeZone('UTC'));
     }
 
-    /**
-     * Get current formatted date and time
-     * Don't use non-GMT time, unless you know the difference and really need to.
-     *
-     * @throws Exception
-     */
-    public static function currentFormattedDateTime(string $format = self::MYSQL_DATE_FORMAT, $gmt = true): string
+    public static function formatForDisplay(DateTime $dateTime): string
```

Comment:
```suggestion
    public static function formatForDisplay(?DateTime $dateTime): string
```

cos used with combination with formatFromDatabaseDateIfNotEmpty

Reply:
Changes implemented.